### PR TITLE
Edit Rapid Antigen Test profile (EXPOSUREAPP-7119)

### PR DESCRIPTION
**Pull request by mtwalli**
- Enable editting
- Open information screen from profile 
- Change bluish gradient

### CODE REVIEW

**Review by ChristinaTsigganos — DISMISSED** [review]
UA approved

**Comment by axelherbstreith** [comment]
While testing, I noticed two things:
- Menu does not adapt to dark mode
- When editing the test profile without changing dob, then save the changes and press again on edit, the dob is not shown anymore

Nothing to complain besides that, looks really nice 💯

**Review by axelherbstreith — DISMISSED** [review]
lgtm! Thanks for fixing 🥇

File: Corona-Warn-App/src/main/res/values/antigen_strings.xml [hunk 1]
```diff
@@ -197,7 +197,11 @@
     <!-- XBUT: Create RAT Qr code profile next button-->
     <string name="rat_qr_code_profile_next_button">"Continue"</string>
     <!-- XTXT: Create RAT Qr code profile delete item-->
-    <string name="rat_qr_code_profile_delete_item">"Remove"</string>
+    <string name="rat_qr_code_profile_delete_item">Schnelltest-Profil entfernen</string>
+    <!-- XTXT: Create RAT Qr code profile edit item-->
+    <string name="rat_qr_code_profile_edit_item">Schnelltest-Profil bearbeiten</string>
+    <!-- XTXT: Create RAT Qr code profile information item-->
+    <string name="rat_qr_code_profile_information_item">Infomation</string>
```
Comment (BMItr) [inline Corona-Warn-App/src/main/res/values/antigen_strings.xml:204]: ->Information

**Review by BMItr — APPROVED** [review]
lvgtm!

**Review by axelherbstreith — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**